stable/locust/Chart.yaml

```diff
@@ -1,5 +1,11 @@
+apiVersion: v1
 name: locust
-description: A chart for locust distributed load testing
-version: 0.1.0
+description: A modern load testing framework
+version: 0.1.1
+maintainers:
+  - name: Vincent De Smet
```

Comment:
oops sorry, I didn't see #881. I made this change because we found it caused https://github.com/helm/monocular/issues/230!

Go ahead and make the changes in #881, I'll close this out :)